app/modules/user_sessions/views/user_sessions_tab.php

```diff
@@ -0,0 +1,32 @@
+<p>
+<table class="user_sessions table table-striped table-bordered">
```

Comment:
It would be nice to generate the table via a JSON call to the server instead of rendering php inline